otx/algorithms/action/adapters/mmaction/core/evaluation/ava_utils.py

```diff
@@ -0,0 +1,125 @@
+"""Collection of AVA evaluiation utils.."""
+
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import logging
+import time
+from collections import defaultdict
+
+import numpy as np
+from mmaction.core.evaluation.ava_evaluation import (
+    object_detection_evaluation as det_eval,
+)
+from mmaction.core.evaluation.ava_evaluation import standard_fields
+from mmaction.core.evaluation.ava_utils import print_time, read_exclusions
+
+
+# pylint: disable=too-many-locals, too-many-branches
+def ava_eval(predictions, result_type, labels, video_infos, exclude_file, verbose=True, custom_classes=None):
```

Comment:
Is `ava` just a dataset format? As a newcomer, the word `ava` is not familiar. Why is eval here separately? Can't it be attached to the Dataset class?
Also, it would be easier to understand if change the name of the function and the file to `ava_dataset` (eg, `ava_utils `-> `ava_dataset_utils`) instead of `ava`.

Reply:
ava_eval function is in exact same path in latest mmaction(mmaction/core/evaluation).
They name this function as ava_eval because mmaction only support AVA dataset for action detection.
However otx support other dataset format too(e.g. UCF, JHMDB), therefore it is needed to change function's name, and add description as "Adapt from ava_eval from original mmaction"
What do you think the path of this file? I put it in this directory to sync with mmaction. I wonder general strategy of otx for adaptation of inner function of mmfamily

Reply:
Hmm... I agree with this location, but others may have ideas. It would be nice to explain at the OTX development meeting and listen to others opinion.